### ISSUE TITLE
Licensing

### ISSUE THREAD

**Issue by KalitaAlexey** [KalitaAlexey/sip_server#1]
I'm using libsip that is MIT.
Where should I include LICENSE for MIT?
@Bytebuddha?

**Comment by patrickisgreene** [comment]
I'm not sure what you mean. It doesn't matter to me and the license specifies you can use however you wish

**Comment by KalitaAlexey** [comment]
> The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.

Given I my library uses your library, but doesn't include it, I shouldn't include LICENSE-MIT anywhere, should I?

**Comment by patrickisgreene** [comment]
Yes that would be correct. It is my understanding that if your using it as a cargo dependency you don't have to worry because it will be copied when building and even if your working from a fork the LICENSE.md file will be included so you should be fine without doing anything.

TL;DR: No you dont have to include it
